src/elements/content-open-with/IconFileMap.js

```diff
@@ -4,13 +4,15 @@
  * @author Box
  */
 import IconAdobeSign from '../../icons/adobe-sign/IconAdobeSign';
+import IconAutoCAD from '../../icons/autocad/IconAutoCAD';
 import IconGoogleDocs from '../../icons/google-docs/IconGoogleDocs';
 import IconGoogleSheets from '../../icons/google-docs/IconGoogleSheets';
 import IconGoogleSlides from '../../icons/google-docs/IconGoogleSlides';
 import FileIcon from '../../icons/file-icon/FileIcon';
 
 const ICON_FILE_MAP = {
     'Adobe Sign': IconAdobeSign,
+    'AutoCAD Web App': IconAutoCAD,
```

Comment:
Is that the actual branding for AutoCAD? Googling shows `web app` in lower case.

Reply:
the branding is actually in EUA when we add menu items, but yeah you are right the wording shall be in low case for consistency, will change it

Reply:
@MonteWood this change is not needed since Auto Cad isn't going into the Open With Element. It needs to be referenced by the internal implementation only.